Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -1167,3 +1157,5 @@ def main():
 
 if __name__ in ('__main__', '__builtin__', 'builtins'):
     main()
+
+register_module_line('IBM Resilient Systems', 'end', __line__())
```

Comment:
done. can you possibly explain what it does because it is in the original Resilient file (i.e. we have not added this ourselves), and the demisto documentation doesn't explain. Thanks.

Reply:
These lines are for making traceback errors look correct, from what I recall. They were automatically added, but its a known issue and should be removed manually for now